docs/timeseries/lombscarglemb.rst

```diff
@@ -0,0 +1,413 @@
+.. _stats-lombscarglemultiband:
+
+********************************************
+Lomb-Scargle Periodograms for Multiband Data
+********************************************
+
+The Lomb-Scargle periodogram (after Lomb [1]_, and Scargle [2]_) is a commonly
+used statistical tool designed to detect periodic signals in unevenly spaced
+observations. The base :class:`~astropy.timeseries.LombScargle` provides an
+interface for several implementations of the Lomb-Scargle periodogram. However,
+:class:`~astropy.timeseries.LombScargle` only handles a single band of data.
+The :class:`~astropy.timeseries.LombScargleMultiband` class adapts this
+interface to handle multiband data (where multiple bands/filters are present).
+
+The code here is adapted from the `astroml`_ package ([3]_, [4]_) and the
+`gatspy`_ package ([5]_, [6]_), but conforms closely to the design paradigms
```

Comment:
Here of course it's fine to leave a reference to astroml and gatspy as in the original LombScargle page.